resources/containers/styles/field-modal.module.css

```diff
@@ -0,0 +1,89 @@
+.modalOverlay {
```

Comment:
THis file will disappear once you refactor

Reply:
What does this mean?

Reply:
Once we use or create the common resources, you wont need the `field-modal.css` because shall not be specific design for this page.